python/doc/usecases/use_case_fireSatellite.rst

```diff
@@ -0,0 +1,272 @@
+
+with :math:`\tau_{tot}` the total torque, :math:`\tau_{slew}` the slewing torque, :math:`\tau_{dist}` the disturbance torque, :math:`\tau_{g}` the gravity gradient torque, :math:`\tau_{sp}` the solar radiation torque, :math:`\tau_{m}` the magnetic filed interaction torque, :math:`\tau_{a}` the aerodynamic torque.
+
+The attitude control power :math:`P_{ACS}` is finally defined by
+
+.. math::
+    P_{ACS} = \tau_{tot}\omega_{max} + n P_{hold}
+
+The power discipline has 16 inputs and computes the total solar array size and total power by,
+
+.. math::
+    P_{tot} = P_{ACS} + P_{other}
+
+.. math::
+    A_{sa} = \frac{P_{sa}}{P_{EOL}}
+
+with,
+
+.. math::
+    P_{EOL} = P_{BOL}(1-\epsilon_{deg})^{LT},
+
+the power production capability at the end of life, defined by 
+
+.. math::
+    P_{BOL} = \eta F_s I_d \cos(i)
+    
+the power production capability at the beginning of life, and 
+
+
+.. math::
+    P_{sa} = \frac{\left( \frac{P_e T_e}{0.6} + \frac{P_d}{T_d}{0.8} \right)}{T_d}
+ 
+is the required power output. :math:`P_e` and :math:`P_d` are the satellite requirements during eclipse and daylight (here :math:`P_e = P_d = P_{tot}`). :math:`T_e` and :math:`T_d` are the time per orbit spent in eclipse and daylight.
+
+Finally, the inertia can be derived as follows,
+
+.. math::
+    I_{min} = \min(I_{totX},I_{totY},I_{totZ}) 
+
+.. math::
+    I_{max} = \max(I_{totX},I_{totY},I_{totZ}) 
+
+with :math:`I_{tot} = I_{sa} + I_{body}` the total moment of inertia in the three dimensions, that depends on, 
+
+.. math::
+    I_{saX} = m_{sa} \left( \frac{1}{12} (L^2+t^2) + (D+\frac{L}{2})^2 \right)
+
+
+.. math::
+    I_{saY} = m_{sa} (t^2+W^2)
+
+.. math::
+    I_{saZ} = m_{sa} \left( \frac{1}{12} (L^2+W^2) + (D+\frac{L}{2})^2 \right)
+
+with :math:`L = \sqrt{\frac{A_{sa} r_{tw}}{n_{sa}}}` the length of the solar array,
+:math:`W = \sqrt{\frac{A_{sa}}{r_{tw}n_{sa}}}` the width of the solar array, and
+:math:`m_{sa} = 2\rho_{sa}L W t` the mass of the solar array.
+   
+Two tunings parameters are present : 
+
+- :math:`tolFPI` : the tolerance on the fixed point iteration algorithm used in the multidisciplinary analysis, 1e-3
+- :math:`maxFPIIter` : the maximum number of iterations of the fixed point iteration algorithm used in the multidisciplinary analysis, 50
+
+References
+----------
+
+* Wertz, J. and Larson, W. (1999) Space Mission Analysis and Design, Microcosm, Inc. Torrance, CA.
+
+
+* Sankararaman, S., Mahadevan, S. (2012). Likelihood-based approach to multidisciplinary analysis under uncertainty. Journal of Mechanical Design, 134(3):031008
+
+
+* Zaman, K. (2012). Modeling and management of epistemic uncetainty for multidisciplinary system analysis and design, PhD thesis, Vanderbilt University, USA
+
+
+Load the use case
+-----------------
+
+We can load this model from the use cases module as follows :
+
+.. code-block:: python
+
+    >>> from openturns.usecases import fireSatelitte_function

```

Comment:
```suggestion
    >>> from openturns.usecases import fireSatelite_function

```
Importing a name like xxxYyy looks strange to me. Couldn't it be something like 
`from openturns.usecases import fire_satelite`?